policy/modules/services/matrixd.te

```diff
@@ -83,6 +83,7 @@ corenet_udp_bind_generic_node(matrixd_t)
 corenet_udp_bind_generic_port(matrixd_t)
 corenet_udp_bind_reserved_port(matrixd_t)
 
+dev_read_sysfs(matrixd_t)
```

Comment:
I'm curious, do you have an AVC for this? Was this seen on Synapse?

Reply:
I only use synapse so that was where it was. I don't have an AVC. Do you run synapse without seeing it?